pinc/DPDatabase.inc

```diff
@@ -59,12 +59,32 @@ final class DPDatabase
         return self::$_connection;
     }
 
-    static public function log_error()
+    static public function escape($value)
```

Comment:
This is a good idea and I'm bummed that it's a few months late. I'm loath to change it at this time. One way to think about this though is that we are escaping in a DPDatabase namespace (since it's a static function) and the only escaping one would do for a database is for SQL.

Reply:
Yeah, I don't think people will get confused using it given the scoped naming.